openfecwebapp/templates/partials/committees-filter.html

```diff
@@ -9,11 +9,11 @@
 <div class="js-accordion accordion--neutral" data-content-prefix="filter" data-open-first="false">
   <div class="filters__inner">
     {{ typeahead.field('q', 'Committee name or ID', False, dataset='committees', allow_text=True) }}
-    {{ years.years('cycle', 'Two-year period') }}
+    {{ years.years('cycle', 'Election year') }}
```

Comment:
Sorry, but this one I think actually _should_ be "Years active" since it's based on the year they have financial activity.